composer.json

```diff
@@ -19,7 +19,7 @@
     "php": "^7.3 || ^8.0",
     "ext-json": ">=1.3.7",
     "ezimuel/ringphp": "^1.1.2",
-    "psr/log": "^1|^2"
+    "psr/log": "^1|^2|^3"
```

Comment:
PSR-2 is [deprecated](https://www.php-fig.org/psr/psr-2/) in favour of [PSR-12](https://www.php-fig.org/psr/psr-12/)

Reply:
I think you're mixing things here, [`psr/log`](https://github.com/php-fig/log) package is about [`PSR-3`](https://www.php-fig.org/psr/psr-3/) which currently has 3 versions.

PSR-2 and PSR-12 are about coding standard

Reply:
@franmomu correct, but it doesnt change the fact that PSR-2 is deprecated

Reply:
so... what should I change in this PR?

Reply:
remove `^2` from line 22 in `composer.json`

Reply:
Please read this again:

> I think you're mixing things here, [`psr/log`](https://github.com/php-fig/log) package is about [`PSR-3`](https://www.php-fig.org/psr/psr-3/) which currently has 3 versions.

`psr/log` has nothing to do with psr-2

Reply:
> `psr/log` has nothing to do with psr-2

Yes. So will close this as misunderstanding. Must have been.